src/bundle/Resources/mappings/symfony-service-tag-name-map.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+return [
+    'doctrine.dbplatform' => 'ibexa.doctrine.db.platform',
+    'ez.limitation.formMapper' => 'ibexa.admin_ui.limitation.mapper.form',
+    'ez.limitation.valueMapper' => 'ibexa.admin_ui.limitation.mapper.value',
+    'ezplatform.admin_ui.component' => 'ibexa.admin_ui.component',
+    'ezplatform.admin_ui.config_provider' => 'ibexa.admin_ui.config.provider',
+    'ezplatform.admin_ui.form_ui_action_mapper' => 'ibexa.admin_ui.form_ui.mapper.action',
+    'ezplatform.admin_ui.siteaccess_preview_voter' => 'ibexa.admin_ui.site_access.preview.voter',
+    'ezplatform.admin_ui.user_setting.form_mapper' => 'ibexa.admin_ui.user_setting.mapper.form',
+    'ezplatform.admin_ui.user_setting.value' => 'ibexa.admin_ui.user_setting.value',
+    'ezplatform.cache_response_tagger' => 'ibexa.cache.http.response.tagger',
+    'ezplatform.calendar.event_source' => 'ibexa.calendar.event.source',
+    'ezplatform.calendar.event_type.future_hide.action' => 'ibexa.calendar.event.type.action.future_hide',
+    'ezplatform.calendar.event_type.future_publication.action' => 'ibexa.calendar.event_type.action.future_publication',
+    'ezplatform.calendar.event_type' => 'ibexa.calendar.event.type',
+    'ezplatform.cron.job' => 'ibexa.cron.job',
+    'ezplatform.ezrichtext.configuration.provider' => 'ibexa.field_type.richtext.configuration.provider',
+    'ezplatform.field_type.comparable.engine' => 'ibexa.field_type.comparable.engine',
+    'ezplatform.field_type.comparable' => 'ibexa.field_type.comparable',
+    'ezplatform.field_type.external_storage_handler.gateway' => 'ibexa.field_type.storage.external.handler.gateway',
+    'ezplatform.field_type.external_storage_handler' => 'ibexa.field_type.storage.external.handler',
+    'ezplatform.field_type.form_mapper.definition' => 'ibexa.admin_ui.field_type.form_mapper.definition',
```

Comment:
```suggestion
    'ezplatform.field_type.form_mapper.definition' => 'ibexa.admin_ui.field_type.form.mapper.definition',
```
Not sure what is the general convention here, but I assume we don't separate core mechanism names like `FieldType`. However, if `EventType` is tagged as `event.type`, I would follow that rule also for mappers.

Reply:
> Not sure what is the general convention here, but I assume we don't separate core mechanism names like `FieldType`. However, if `EventType` is tagged as `event.type`, I would follow that rule also for mappers.

Makes sense :+1: